src/libstore/ssh.cc

```diff
@@ -41,7 +41,7 @@ void SSHMaster::addCommonSSHOpts(Strings & args)
 {
     auto state(state_.lock());
 
```

Comment:
Unsure if we should actually print the value of NIX_SSHOPTS here, can it contain secrets? But anyway the point is using `addTrace`.

```suggestion
  std::string env = getEnv("NIX_SSHOPTS").value_or("");
  try {
    std::list<std::string> parts = shellSplitString(env);
  } catch (Error & e) { 
     e.addTrace({}, "while parsing NIX_SSHOPTS '%s'", env);
     throw; 
 } 
```